Extensions/BBText/JsExtension.js

```diff
@@ -0,0 +1,588 @@
+        'JsPlatform/Extensions/bbcode32.png',
+        objectBBText
+      )
+      .setIncludeFile('Extensions/BBText/bbtextruntimeobject.js')
+      .addIncludeFile('Extensions/BBText/bbtextruntimeobject-pixi-renderer.js')
+      .addIncludeFile(
+        'Extensions/BBText/pixi-multistyle-text/dist/pixi-multistyle-text.umd.js'
+      );
+
+    // Utility function to add both a setter and a getter to a property from a list. Useful for setting multiple generic properties
+    const addSettersAndGettersToObjectHelper = (
+      gdObject,
+      properties,
+      objectName
+    ) => {
+      properties.forEach(property => {
+        const parameterType =
+          property.type === 'boolean' ? 'yesorno' : property.type;
+
+        // Add all the generic GETTERS
+        if (parameterType === 'number') {
+          gdObject
+            .addExpression(
+              `Get${property.functionName}`,
+              _('Get ') + property.description,
+              _('Get ') + property.description,
+              _(''),
+              '',
+              `${property.iconPath}.png`,
+              `${property.iconPath}.png`
+            )
+            .addParameter(
+              'object',
+              _(`${objectName} object`),
+              objectName,
+              false
+            )
+            .getCodeExtraInformation()
+            .setFunctionName(`get${property.functionName}`);
+        } else {
+          gdObject
+            .addStrExpression(
+              `Get${property.functionName}`,
+              _('Get ') + property.description,
+              _('Get ') + property.description,
+              _(''),
+              '',
+              `${property.iconPath}.png`,
+              `${property.iconPath}.png`
+            )
+            .addParameter(
+              'object',
+              _(`${objectName} object`),
+              objectName,
+              false
+            )
+            .getCodeExtraInformation()
+            .setFunctionName(`get${property.functionName}`);
+        }
+
+        // Add all the generic SETTERS
+        if (parameterType === 'number' || parameterType === 'string') {
+          const expressionType =
+            parameterType === 'number' ? 'expression' : 'string';
+          gdObject
+            .addAction(
+              `Set${property.functionName}`,
+              _('Set ') + property.description,
+              _('Set ') + property.description,
+              _('Do _PARAM1__PARAM2_ to the ') +
+                property.description +
+                _(' of _PARAM0_'),
+              '',
+              `${property.iconPath}.png`,
+              `${property.iconPath}.png`
+            )
+            .addParameter(
+              'object',
+              _(`${objectName} object`),
+              objectName,
+              false
+            )
+            .addParameter('operator', _("Modification's sign"), '', false)
+            .addParameter(expressionType, _(property.paramLabel), '', false)
```

Comment:
`paramLabel` is already the result of calling the translation function on some string: `_()`

As a rule of thumb, never ever use `_()` on a variable.

Reply:
(also check the rest of the file for these)

Reply:
ah sorry, not sure how that ended like that xD I should be more careful when copy  pasting